experiments/ClimaEarth/setup_run.jl

```diff
@@ -231,6 +231,22 @@ function setup_and_run(config_dict::AbstractDict)
     Random.seed!(random_seed)
     @info "Random seed set to $(random_seed)"
 
+    isnothing(restart_t) && (restart_t = Checkpointer.t_start_from_checkpoint(dir_paths.checkpoints))
+    isnothing(restart_dir) && (restart_dir = dir_paths.checkpoints)
+    if !isnothing(restart_t)
+        if t_start isa ITime
+            t_start, _ = promote(ITime(restart_t), t_start)
+        else
+            t_start = maybe_t_start_from_checkpoint
```

Comment:
I don't see `maybe_t_start_from_checkpoint` defined anywhere

Reply:
I removed all the `auto_restart` infra because I can handle differently, in a simpler way